aviary/subsystems/propulsion/test/test_propeller_performance.py

```diff
@@ -76,11 +76,6 @@ def setUp(self):
                               800 * np.ones(num_nodes), units="ft/s")
         pp.set_input_defaults(Dynamic.Mission.VELOCITY,
                               100. * np.ones(num_nodes), units="knot")
-        num_blades = 4
```

Comment:
Not sure why these set_vals are removed from this test.

Reply:
duplicates with below